docs/APIRef.DeviceObjectAPI.md

```diff
@@ -320,7 +320,7 @@ Check out Detox's [own test suite.](../detox/test/e2e/06.device-orientation.test
 
 Sets the simulator/emulator location to the given latitude and longitude.
 
-> On iOS `setLocation` is dependent on [fbsimctl](https://github.com/facebook/idb/tree/4b7929480c3c0f158f33f78a5b802c1d0e7030d2/fbsimctl) which [is now deprecated](https://github.com/wix/Detox/issues/1371). If `fbsimctl` is not installed, the command will fail, asking for it to be installed.
+> On iOS `setLocation` is dependent on [applesimutils@546785](https://github.com/wix/AppleSimulatorUtils/commit/5467857407da4653305af119b2486051664a4f97) or more recent, that has `--setLocation` support.
```

Comment:
Should we be removing fbsimctl installation for any setup guide as well, then?

Reply:
There are no other mentions of `fbsimctl`, AFAIC.